test/helpers/vmware/intg_test_context.go

```diff
@@ -109,16 +111,23 @@ func NewIntegrationTestContextWithClusters(goctx context.Context, integrationTes
 	})
 
 	var config *rest.Config
+
+	// get crd paths
+	_, filename, _, ok := goruntime.Caller(0)
+	Expect(ok).To(BeTrue(), "Failed to get information for current file from runtime")
+	root := path.Join(path.Dir(filename), "..", "..")
+	crdPaths := []string{
+		filepath.Join(root, "config", "default", "crd"),
```

Comment:
Looks like usually we use config/default/crd/bases

This probably means that these CRDs are not required for this test. Can you try dropping this line and then checking if the test still works?

Reply:
When I removed the CRDs from here, and the `CRDDirectoryPaths` field in `envtest.Environment`, all tests passed successfully.

Reply:
I also took a closer look. I confirmed that the test is actually executed (so we don't need a follow-up for that).

It just looks like we really don't need the CRDs. I think this is because this func creates an env test environment (apiserver+etcd) to simulate a workload cluster (or in supervisor mode called a guest cluster).

On these clusters we simply don't need the CAPV CRDs.

So I would suggest lets drop the CRD deployment here (~l.115-122 + l.130)